### PR TITLE
WIP: Feature: Library logs history

### PR DESCRIPTION
**Pull request by mithunbharadwaj**

### CODE REVIEW

File: asab/library/service.py [hunk 1]
```diff
@@ -73,6 +73,7 @@ def __init__(self, app, service_name, paths=None):
 		super().__init__(app, service_name)
 		self.Libraries = list()
 		self.Disabled = {}
+		self.History = []
```
Comment (ateska) [inline asab/library/service.py:76]: Don't store history in the microservice, always load that fresh.

**Review by ateska — CHANGES_REQUESTED** [review]
I assume that the write history function will follow.

**Comment by ateska** [comment]
Oh, this whole functionality needs to be in the `asap-library` not here.

**Comment by ateska** [comment]
Will this be closed? @mithunbharadwaj

**Comment by mithunbharadwaj** [comment]
Closing MR, Solved the problem in Microservice.

**Pull request closed without merging**